src/SDKs/CognitiveServices/dataPlane/Vision/Vision/Microsoft.Azure.CognitiveServices.Vision.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Vision</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Vision APIs.</Description>
-    <VersionPrefix>1.0.1-preview</VersionPrefix>
+    <VersionPrefix>1.0.2-preview</VersionPrefix>
```

Comment:
@lebronJ please model your csproj according to the below project.
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/CognitiveServices/dataPlane/Search/BingCustomSearch/BingCustomSearch/Microsoft.Azure.CognitiveServices.Search.CustomSearch.csproj
you are missing imports at the bottom of the project file and also you don't need few project properties that you have added.

Reply:
Refined this csproj following the styles of above `BingCustomSearch`